drivers/soundwire/intel.c

```diff
@@ -1590,12 +1600,23 @@ static int intel_resume_runtime(struct device *dev)
 		return ret;
 	}
 
+	/*
+	 * master is powered off on intel platforms for clock stop state,
+	 * so it should be reset for master initialization.
+	 */
```

Comment:
No. exit-reset performs a HardReset which is not required when resuming from clock stop mode.
If you do a hard-reset, you force all slaves to re-enumerate and be re-initialized, so you lose all benefits of the clock stop mode.
You will still be able to resume, but it's much longer than needed, and you've lost all context.
You need to make this conditional on clock-stop being used